spec/models/dialog_field_sorted_item_spec.rb

```diff
@@ -178,49 +178,76 @@
 
     context "when the field is not dynamic" do
       let(:dynamic) { false }
-      let(:default_value) { "abc" }
 
-      context "when the field is required" do
-        let(:required) { true }
+      context "when the data type is not integer" do
+        context "when the default_value is set" do
+          let(:default_value) { "abc" }
 
-        it "returns the values with the first option being a nil 'Choose' option" do
-          expect(dialog_field.values).to eq([[nil, "<Choose>"], %w(test test), %w(abc abc)])
+          context "when the field is required" do
+            let(:required) { true }
+
+            it "returns the values without the first option being a nil option" do
+              expect(dialog_field.values).to eq([%w(test test), %w(abc abc)])
+            end
+          end
+
+          context "when the field is not required" do
+            it "returns the values with a nil 'None' option" do
+              expect(dialog_field.values).to eq([[nil, "<None>"], %w(test test), %w(abc abc)])
```

Comment:
@eclarizio - I'm assuming this part ( that the `[nil, '<None>']` as the first option is the bit that resolves the main description of BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1462375